tests/models/wav2vec2/test_modeling_wav2vec2.py

```diff
@@ -1151,6 +1151,42 @@ def get_logits(model, input_features):
 
         self.assertTrue(torch.allclose(logits, logits_2, atol=1e-3))
 
+    def test_load_and_new_target_lang_adapter(self):
```

Comment:
might be good to add a short comment to explain a bit what this test does, as this is for a very special situation.

Reply:
even just a link to issue and this PR page is fine for me.